src/popoverManager.ts

```diff
@@ -67,12 +67,12 @@ export class PopoverManager<T> extends Component {
         this.lastEvent = null;
     }
 
-    hide(lazy: boolean = false) {
+    hide(lazy = false) {
```

Comment:
I agree that we don't need explicit typing here, but I don't see why it can cause any errors. Can I ask why this change is needed? Do you see any compilation errors?

Reply:
Oh is it just a cleanup? Then it makes sense.

Reply:
To be honest in my opinion it is more readable this way, also for me the IDE threw an error for it so that's why I thought I should remove it.

Reply:
I believe that's not an "actual" error but a styling error from an eslint rule: https://typescript-eslint.io/rules/no-inferrable-types/

Personally, I feel an explicit typing helps keep consistency with other parameter declarations such as `event: UserEvent | null = null`.
So I will update `.eslintrc` rather than changing the code.